setup.py

```diff
@@ -286,6 +286,7 @@ def write_version(filename: str = os.path.join(*[dirname(__file__), "airflow", "
     'pymssql~=2.1.1',
 ]
 mysql = [
+    'mysql-connector-python==8.0.18',
```

Comment:
And should we pin the main version instead of pin specific version, I found out it release frequently。https://pypi.org/project/mysql-connector-python/#history

Reply:
8.0.19 does not work: https://github.com/apache/airflow/pull/6576#issuecomment-591519540

Reply:
```suggestion
    'mysql-connector-python>=8.0.0,mysql-connector-python<8.0.19',
```

Reply:
> 8.0.19 does not work: [#6576 (comment)](https://github.com/apache/airflow/pull/6576#issuecomment-591519540)

Oh, get it.

Reply:
Should we dig in to find out why or just limit version to 8.0.18?

Reply:
> According to https://dev.mysql.com/doc/relnotes/connector-python/en/news-8-0.html 8.0.11 was the first release. WDYT?

```suggestion
    'mysql-connector-python>=8.0.11, <=8.0.18',
```
Ignore the package name should be better